assets/csidriveroperators/README.md

```diff
@@ -36,4 +36,4 @@ combined to generate the resulting assets, which are dumped to the `generated`
 subdirectory in the `standalone`, `hypershift/guest` and `hypershift/mgmt`
 directories.
 
-[hcp]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html-single/hosted_control_planes/index
+[hcp]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html-single/hosted_control_planes/index
```

Comment:
IMHO `latest` works too:

```suggestion
[hcp]: https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html-single/hosted_control_planes/index
```

Reply:
I believe this only works until `hosted_control_planes/index` changes its URL or anchor.  That being said I can definitely change it if you prefer `latest`.